lib/literature/cloudflare.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Literature.Cloudflare do
+  @moduledoc """
+    Literature Cloudflare
+  """
+
+  alias Literature.Config
+
+  @doc """
+  Purge cloudflare files.
+  """
+  def purge_cloudflare_files(socket, slug) do
+    if Config.cloudflare_config() do
+      "#{Config.cloudflare_api_host()}/zones/#{Config.cloudflare_identifier()}/purge_cache"
+      |> HTTPoison.post(build_json(socket, slug), [
+        {"X-Auth-Email", Config.cloudflare_email()},
+        {"X-Auth-Key", Config.cloudflare_api_key()},
+        {"Content-Type", "application/json"}
+      ])
+    else
+      :ok
```

Comment:
@christianlj might want to raise an error here to set cloudflare configuration

Reply:
Hi @dgross881, we can't force users to use Cloudflare so we can't raise an error here.